test/cmd/templates.sh

```diff
@@ -11,11 +11,13 @@ os::log::install_errexit
 # This test validates template commands
 
 oc get templates
-oc create -f examples/sample-app/application-template-dockerbuild.json
+oc create -f examples/sample-app/application-template-dockerbuild.json 
+oc create -f examples/sample-app/application-template-dockerbuild.json -n openshift
```

Comment:
Hrm, someone changed that under me then.

On Wed, Sep 30, 2015 at 3:53 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In test/cmd/templates.sh
> <https://github.com/openshift/origin/pull/4863#discussion_r40795419>:
>
> > @@ -11,11 +11,13 @@ os::log::install_errexit
> >  # This test validates template commands
> >
> >  oc get templates
> > -oc create -f examples/sample-app/application-template-dockerbuild.json
> > +oc create -f examples/sample-app/application-template-dockerbuild.json
> > +oc create -f examples/sample-app/application-template-dockerbuild.json -n openshift
>
> pretty sure that's not true... in test-cmd.sh:
>
>   # switch back to a standard identity. This prevents individual tests from changing contexts and messing up other tests
>   oc project ${CLUSTER_ADMIN_CONTEXT}
>
> subtests start as cluster admin in an isolated project and can change
> users if they want
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4863/files#r40795419>.
>